### PR TITLE
Universe: referenced file had the wrong name

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
## Testing

Hard to test to be honest. We had the wrong name in the S3 bucket file than the one referencing in the json file. Please take a look at https://s3.console.aws.amazon.com/s3/buckets/dcos-ui-universe/oss/dcos-ui/latest/?region=us-west-2&tab=overview

## Trade-offs

- Choose to do proper versions in a next PR

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -105,7 +105,7 @@ pipeline {
         ]) {
           sh "git clone https://github.com/mesosphere/dcos-commons.git ../dcos-commons"
           sh "tar czf oss-build.tar.gz dist"
```
Comment (juliangieseke) [inline Jenkinsfile]: 🤔

Comment (DanielMSchmidt) [reply Jenkinsfile]: not really, but then again there is the other version branch fixing this. either we merge this now or close in favor of real versions.

**Comment by nLight** [comment]
Was that superseded by https://github.com/dcos/dcos-ui/pull/3388

**Comment by DanielMSchmidt** [comment]
Yes, but I would rather have this merged first to get something working out there.

**Pull request merged**